typesafety/test_maybe/test_maybe_type_cast.yml

```diff
@@ -22,7 +22,7 @@
   main: |
     from returns.maybe import Some
 
-    reveal_type(Some(1))  # N: Revealed type is "returns.maybe.Maybe[builtins.int*]"
+    reveal_type(Some(1))  # N: Revealed type is "returns.maybe.Some[builtins.int*]"
```

Comment:
@sobolevn I'd like to know your opinion about this change!

Reply:
I think that we don't have any other options, because as far as I know, this cannot be changed even with `__new__` annotation. But, can you please try to be sure?

Reply:
I've just tried, haven't worked! 😢